.env.example

```diff
@@ -5,5 +5,5 @@ WEBHOOK_SECRET=development
 # Uncomment this to get verbose logging
 # LOG_LEVEL=trace # or `info` to show less
 
-# Subdomain to use for localtunnel server. Defaults to your local username.
-# SUBDOMAIN=
+# Go to https://smee.io/new
```

Comment:
Do you think it's worth making this more specific? Like

> Go to https://smee.io/new use the URL you're redirected to

Or maybe we should show a welcome message on Smee if a user comes from /new to tell em what's what?